### DISCUSSION TITLE
Where can I download your old builds of the program?

### DISCUSSION THREAD

**Discussion by niksavc** [character-map-uwp/Character-Map-UWP#249] [Q&A] [answered]
I understand that you only engage in Microsoft Store, but can download old builds?
![image](https://user-images.githubusercontent.com/58150480/227780300-ee619b55-b0b6-44e3-93e8-76b467af1d1c.png)

**Comment by JohnnyWestlake** [comment]
You would have to build it from the source code yourself, but if you want that style it's still available in Settings -> Look & Feel ->  Design theme -> Windows 10 -> Restart App

**Comment by niksavc** [comment]
I can't install the app, itself because I have Win 10 1909, and there Win 10/11 20h2, can at least invest in archive.org, or on the actual github.

**Comment by EdiWang** [comment, marked as answer]
Microsoft Store doesn't allow users to download old versions of an App. You need to clone the repo, checkout a git commit from old version you want, and build it yourself.
